Comanche055/LUNAR_AND_SOLAR_EPHEMERIDES_SUBROUTINES.agc

```diff
@@ -69,44 +62,43 @@
 #
 #	1) SPECIFIED GROUND ELAPSED TIME IN CS x B-28 LOADED IN MPAC.
 #
-#	2) TIMEMO -- TIME AT THE CENTER OF THE RANGE OVER WHICH THE LUNAR
+#	2) TIMEMO - TIME AT THE CENTER OF THE RANGE OVER WHICH THE LUNAR
```

Comment:
How should we handle sentence/paragraph continuation? Should we leave it be like this or should we put a tab after the number, e.g. `1)⇥` like in https://github.com/chrislgarry/Apollo-11/blob/master/Comanche055/P76.agc? 🤔

Reply:
Good question. I would probably use the format in that file since it is closer to how it looked in the printouts.

Reply:
I didn't change those  '--' in [Proof P76 (#239)](https://github.com/chrislgarry/Apollo-11/pull/317)

Reply:
@oldmud0 Hmm. If that's the argument, would it be even closer if we just put 3 spaces in the continuation?

<img width="570" alt="screen shot 2018-03-12 at 11 33 17 pm" src="https://user-images.githubusercontent.com/4405796/37313457-9df828c8-264f-11e8-9677-c70038106836.png">

@keller0 🙋‍♂️  I'm actually also kind of undecided about that. Should we use hyphen (-) or double hyphen a.k.a. en dash (--)? Furthermore, in [P76](https://github.com/chrislgarry/Apollo-11/blob/master/Comanche055/P76.agc), I notice that every full stop (.) is always followed by double spaces -- is it a general rule that we have to follow here?

cc: @chrislgarry @wopian

Reply:
@dwiyatci 

* Single space for new words
* Double space for new sentences
* Triple space for indentations

Moving forward, lets change double hyphens to single hyphens to be consistent with the prints. @keller0 its no problem. For new PRs lets do single hyphens unless the printout says otherwise.

Reply:
@chrislgarry Roger that. 🙇